FrontEnd/src/components/HeaderFooter/header/navbar/SearchBox.jsx

```diff
@@ -1,15 +1,29 @@
 import css from "./SearchBox.module.css"
 import icon_search from './search-icon.svg';
+import React, { useState } from "react";
```

Comment:
There is no need to import `React` starting from version 17. Also, good practice is to place import of react hooks at the top of other imports